src/setupTests.js

```diff
@@ -0,0 +1,5 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
+// allows you to do things like:
+// expect(element).toHaveTextContent(/react/i)
+// learn more: https://github.com/testing-library/jest-dom
+import '@testing-library/jest-dom';
```

Comment:
- [ ] Please note that all the boilerplate that got created by the CRA (texts, images, and styles) ought to have gotten deleted.💪🏽 For example:
- App.test.js should be deleted.
- setupTests.js
- reportWebVitals.js
- The react logo images on /public
- If there is CSS code you didn't write, please remove it

Reply:
I wasn't sure if those needed to be deleted too, I'll do right away! 😉